docs/install.md

```diff
@@ -45,14 +45,44 @@ conda install pytorch=1.3.1 cudatoolkit=9.2 torchvision=0.4.2 -c pytorch
 If you build PyTorch from source instead of installing the prebuilt pacakge,
 you can use more CUDA versions such as 9.0.
 
-c. Clone the mmdetection repository.
+c. Install mmcv, you can choose either to compile mmcv from source by the following command
+
+```
+git clone https://github.com/open-mmlab/mmcv.git
+cd mmcv
+pip install -e .
+cd ..
+```
+
+Or you can install the pre-build mmcv.
```

Comment:
Since MMCV has already provided a full list/table of installation, we may directly refer the users to the page of mmcv and simply provide an example here.